frontend/src/components/GovtMasthead/GovtMasthead.tsx

```diff
@@ -52,6 +52,7 @@ const HeaderBar = ({
     textStyle: { base: 'legal', md: 'caption-2' },
     display: 'flex',
     width: '100%',
+    alignItems: 'center',
```

Comment:
this shouldn't be needed I think?

Reply:
thanks! yes, this has been reverted in `436736b`